src/test/java/uk/gov/crowncommercial/dts/scale/cat/controller/ProjectsControllerTest.java

```diff
@@ -192,15 +190,17 @@ void updateProcurementProjectName_200_OK() throws Exception {
   @Test
   void listProcurementEventTypes_200_OK() throws Exception {
 
+    var expectedJson =
```

Comment:
Formatting again - if I apply the formatter it changes (e.g. wraps lines at 100 chars)